pylint/lint/pylinter.py

```diff
@@ -914,12 +912,12 @@ def get_ast(
                 data, modname, filepath
             )
         except astroid.AstroidSyntaxError as ex:
-            # pylint: disable=no-member
+            error, line, col = getattr(ex, "error", None), 0, 0
```

Comment:
Yeah astroid errors are pretty messy, they would definitely benefit from being less dynamic.

Reply:
As we released 2.12.0, should we merge this without an astroid upgrade ? We can fix the message in a follow-up PR later, right ?

Reply:
Yeah. Can we use `mypy` ignores and a comment about it though?

Reply:
There's no mypy ignore to add we're fixing the issue mypy was warning us about 😄 
```suggestion
            # TODO 2.16: Clearer less dynamic Exception classes in astroid
            error, line, col = getattr(ex, "error", None), 0, 0
```

Reply:
Yeah but let's not use getattr. There is no need for it as long as make this exception like we are supposed to in asteroid

Reply:
Hmm, but if we're not using ``get_attr`` we're getting a crash it's what we want to avoid. We're not making the exception like we're supposed to, right now. Maybe something that should be fixed in 2.12.1, instead of arguing about it, I see Jacob is already fixing crashes like a machine and preparing a release.